pom.xml

```diff
@@ -6,7 +6,7 @@
 		mvn -DskipTests -Dbuild.number=71 -DGitBranch=develop help:system
 		
 		# fast build
-		mvn -DskipTests -Dbuild.number=71 -DGitBranch=develop package -o
```

Comment:
Seems like we should have some sort of default for build.number=SNAPSHOT or something like that.. so we know it's not an official release/build version?